packages/babel-plugin-emotion/src/core-macro.js

```diff
@@ -29,7 +34,40 @@ export const transformCssCallExpression = ({
   }
 }
 
-export let coreCssTransformer = ({
+export const transformInlineCsslessExpression = ({ state, babel, path }) => {
+  let t = babel.types
+  let expressionPath = path.get('value.expression')
+  let sourceMap =
+    state.emotionSourceMap && path.node.loc !== undefined
+      ? getSourceMap(path.node.loc.start, state)
+      : ''
+
+  expressionPath.replaceWith(
+    t.callExpression(
+      // the name of this identifier doesn't really matter at all
+      // it'll never appear in generated code
+      t.identifier('___shouldNeverAppearCSS'),
+      [path.node.value.expression]
+    )
+  )
+
+  transformCssCallExpression({
+    babel,
+    state,
+    path: expressionPath,
+    sourceMap
+  })
+
+  const { importSource = '@emotion/core', cssExport = 'css' } = state
```

Comment:
seems like those should be configurable for import mapping case, any proposals for the API?

Reply:
Require that modules that export `Global` also export `css`, this is already done for `jsx`.

Reply:
Done